### PR TITLE
Print Entire Docker output to window

### PR DESCRIPTION
**Pull request by drewgonzales360**
The TLDR is that the content will not print as if someone had called `docker images` or `docker container ls`

This'll follow the standard fzf rules. You could start typing and if you match _anything_ in the line, it'll use whatever value is in the first column

Screen Shot 2019-12-18 at 11 33 13 AM
Screen Shot 2019-12-18 at 11 33 44 AM
Screen Shot 2019-12-18 at 11 35 01 AM

### CODE REVIEW

**Comment by drewgonzales360** [comment]
@pierpo Could you take a peek at this when you get a chance?

**Comment by pierpo** [comment]
Hi! Good thing that you pinged me, I did not see your PR :blush: 

Looks good! Could you provide a screenshot of the new behaviour?

I'll try this out when I can.

Thank you very much for the contribution!

File: fzf-docker.plugin.zsh [hunk 1]
```diff
@@ -17,13 +17,17 @@ _fzf_complete_docker() {
...
+      docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}\t{{.ID}}\t{{.CreatedSince}}"
     )
-  elif [[ $ARGS == 'docker start'* || $ARGS == 'docker restart'* || $ARGS == 'docker stop'* || $ARGS == 'docker rm'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* ]]; then
-    _fzf_complete "--multi --reverse" "$@" < <(
-      docker ps --format '{{.Names}}'
+  elif [[ $ARGS == 'docker rmi'* ]]; then
+    _fzf_complete "--multi --header-lines=1" "$@" < <(
+      docker images --format "table {{.ID}}\t{{.Repository}}\t{{.Tag}}\t{{.Size}}"
+    )
+  elif [[ $ARGS == 'docker stop'* || $ARGS == 'docker rm'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* ]]; then
```
Comment (pierpo) [inline fzf-docker.plugin.zsh]: Why did you remove `start` and `restart`?

**Review by pierpo — APPROVED** [review]
Just one question otherwise the rest looks way better. Thanks a lot for contributing 😊

File: fzf-docker.plugin.zsh (hunk 1 above)
Comment (drewgonzales360) [reply fzf-docker.plugin.zsh]: Ah whoops, just threw it back in.

**Comment by drewgonzales360** [comment]
@pierpo Totally forgot to respond lmao. I added start and restart back into the mix. You can merge this whenever you're ready.

**Comment by pierpo** [comment]
> @pierpo Totally forgot to respond lmao. I added start and restart back into the mix. You can merge this whenever you're ready.

Thank you 😊 I'll try it out today and probably merge if everything's alright!

**Comment by pierpo** [comment]
Fantastic improvement, thanks 😊
